packages/odyssey-storybook/src/contributing/CustomTheme.stories.tsx

```diff
@@ -109,3 +108,43 @@ export const RadioGroupStory: StoryFn = () => {
 };
 
 RadioGroupStory.storyName = "RadioGroup";
+
+export const CustomComponentStory: StoryFn = () => {
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const themeOverrides: ThemeOptions = {
+    components: {
+      MuiListItemText: {
+        styleOverrides: {
+          root: {
+            color: "red",
+          },
+        },
+      },
+    },
+  };
+  const odysseyTheme: OdysseyTheme = createOdysseyTheme();
+  const customOdysseyTheme = useMemo(
+    () =>
+      themeOverrides && createTheme(deepmerge(odysseyTheme, themeOverrides)),
+    [odysseyTheme, themeOverrides]
+  );
```

Comment:
We shouldn't be adding an eslint-disable. Also, I think it's not doing anything.

Reply:
there was typescript error, but fixed it using useMemo